src/main/scala/ch/epfl/ts/engine/MarketFXRules.scala

```diff
@@ -0,0 +1,61 @@
+package ch.epfl.ts.engine
+
+import ch.epfl.ts.data.Currency._
+import ch.epfl.ts.data.{ DelOrder, LimitAskOrder, LimitBidOrder, MarketAskOrder, MarketBidOrder, Order, Streamable, Transaction }
+import ch.epfl.ts.component.fetch.MarketNames
+import ch.epfl.ts.data.MarketBidOrder
+import ch.epfl.ts.data.MarketAskOrder
+
+/**
+ * represents the cost of placing a bid and market order
+ */
+case class CommissionFX(limitOrderFee: Double, marketOrderFee: Double)
+
+/**
+ * Market Simulator Configuration class. Defines the orders books priority implementation, the matching function and the commission costs of limit and market orders.
+ * Extend this class and override its method(s) to customize Market rules for specific markets.
+ *
+ */
+class MarketFXRules {
```

Comment:
This class can probably extend another base class rather than standing on its own.